src/permissions.js

```diff
@@ -286,7 +286,34 @@ export default {
 			// we look up the core product and ensure all the permissions match the id associated.
 			const coreProducts = await helper.cacheCoreProduct(req.query.resetCache, req.authGroup);
 			const coreOrg = await helper.cachePrimaryOrg(req.query.resetCache, req.authGroup);
+
 			if(!coreProducts.length) throw new Error('Could not identify core products for this authgroup');
+
+			// create a non-contextual product list from the token to see if they have admin access to UE Auth through any org
```

Comment:
While the UE Auth product itself may not be the what is requested in terms of access, we do need to know if the user has any access at all.